### PR TITLE
Steps: add center property, resolve #2315

### LINKED ISSUE #2315: Provide a property to center whole Steps component (or do it by default)
Actually, `<el-steps align-center>` only aligns the titles, but the whole steps component is left aligned.

As you can see in this example **http://jsfiddle.net/wc1ef0zz/** the `el-step__icon` is left-aligned, so the whole component is turned to the left. It doesn't looks good on certain cases where you need it centered.

A hacky way I'm trying at the moment is `.el-steps.is-horizontal { padding-left: 20% }` but... this is really ugly :smile:

### LINKED ISSUE #2513: 那个upload中的那个beforeUpload方法里面，我在那边修改了上传的参数之后，发现上传的参数没有被改变
```javascript
 beforeUpload(file) {
      var parentNode = this.uploadTargetNode || this.currentNode;

      this.uploadParams = {
        parentId : parentNode['sys:node-uuid']
      };

      return true;
    },
```

### PR DESCRIPTION
**Pull request by QingWei-Li**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by jikkai** [comment]
提前修复还没有的2513……是#2315 吧🤣

**Comment by QingWei-Li** [comment]
@jikkai 🌚

File: examples/docs/en-US/steps.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ Vertical step bars.
 | process-status | status of current step | string | wait/process/finish/error/success | process |
 | finish-status | status of end step | string | wait/process/finish/error/success | finish |
 | align-center | whether step description is centered | boolean | — | false |
+| center | center whole `Steps` component | boolean | false, true | false |
```
Comment (Leopoldthecoder) [inline examples/docs/en-US/steps.md]: | center | whether Step is centered | boolean | — | false |

File: examples/docs/zh-CN/steps.md [hunk 2]
```diff
@@ -114,6 +114,7 @@
 | process-status | 设置当前步骤的状态 | string | wait/process/finish/error/success | process |
 | finish-status | 设置结束步骤的状态 | string | wait/process/finish/error/success | finish |
 | align-center | 标题描述居中对齐 | boolean | false, true | false |
+| center | 组件居中显示 | boolean | false, true | false |
```
Comment (Leopoldthecoder) [inline examples/docs/zh-CN/steps.md]: | center | 组件是否居中显示 | boolean | —  | false |

（上面一行的 false, true 也改成 — 吧= =）

**Pull request merged**